docs/manual/config/screens.rst

```diff
@@ -52,7 +55,18 @@ entire window.
 
 .. note::
     In X11 backends, transparency will be disabled in a bar if the ``background``
-    color is fully opaque.
+    color is fully opaque. This is primarily to prevent a rendering issue with
+    the ``Systray`` widget where icons will be displayed with a fully transparent
+    background, regardless of the colour of the bar/widget.
+
+    X11 users can also use "pseudotransparency" by setting ``fake_transparency=True``
```

Comment:
Could we name the option `x11_fake_transparency`?

Reply:
Yes.